apps/coordinator/src/components/Wallet/TransactionsTab/FeeBumping/components/FeeBumpSteps/ConfigurationStep/RBF/RBFForm.tsx

```diff
@@ -99,37 +99,19 @@ export const RBFForm: React.FC = () => {
     }
 
     try {
-      let result: FeeBumpResult;
       const txVsize = transaction.vsize || transaction.size;
       const estimatedNewFee = Math.ceil(txVsize * feeBumpRate).toString();
-
-      if (rbfType === RBF_TYPES.CANCEL) {
-        const cancelPsbt = createCancelRBF(feeBumpRate, cancelAddress);
-
-        result = {
-          psbtBase64: cancelPsbt,
-          newFee: estimatedNewFee,
-          newFeeRate: feeBumpRate,
-          strategy: selectedStrategy!,
-          isCancel: true,
-          createdAt: new Date().toISOString(),
-        };
-      } else {
-        // Handle accelerated RBF case
-        const acceleratedPsbt = createAcceleratedRBF(
-          feeBumpRate,
-          changeAddress,
-        );
-
-        result = {
-          psbtBase64: acceleratedPsbt,
-          newFee: estimatedNewFee,
-          newFeeRate: feeBumpRate,
-          strategy: selectedStrategy!,
-          isCancel: false,
-          createdAt: new Date().toISOString(),
-        };
-      }
+      const generatePsbt =
+        rbfType === RBF_TYPES.CANCEL ? createCancelRBF : createAcceleratedRBF;
+
+      const result: FeeBumpResult = {
+        psbtBase64: generatePsbt(feeBumpRate, changeAddress),
```

Comment:
does this need to be changeAddress or cancelAddress? (sorry i wrote my suggestion a little hastily)